package.json

```diff
@@ -10,6 +10,7 @@
   },
   "scripts": {
     "build": "scripts/build.sh",
+    "ci": "npm run lint && npm test && npm run coverage",
```

Comment:
I think the output to xunit for circle's built in reporting was a good addition and I'd like to see it back. @hx-markterry, what are your thoughts?